Documentation/ART_NOTES.md

```diff
@@ -0,0 +1,112 @@
+
+# **Art Notes**
```

Comment:
There's two ART_NOTES files...
I'm guessing this one is the right one?

Reply:
Oh weird, I'll delete the duplicate!